packages/@dcl/ecs/package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@dcl/crdt": "^1.0.0-20220819145912.commit-35069cd",
     "@dcl/js-runtime": "file:../js-runtime",
-    "@dcl/protocol": "^1.0.0-3524279049.commit-02b9080"
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-3602487709.commit-3dbbf8f.tgz"
```

Comment:
TODO: change dependency after https://github.com/decentraland/protocol/pull/86 is merged